### DISCUSSION TITLE
Keyboard Shortcut for Notification Center

### DISCUSSION THREAD

**Discussion by ghost** [linuxdeepin/developer-center#3148] [Features Request & Ideas | 特性请求 & 头脑风暴]
Would be nice to have in Keyboard Shortcuts option to set a shortcut for opening Notification Center

**Comment by BLumia** [comment]
Currently you can add a new shortcut in the Control Center that execute the following command:

```
dbus-send --type="method_call" --dest=com.deepin.dde.osd /org/freedesktop/Notifications com.deepin.dde.Notification.Show
```

Adding a default shortcut could be useful, I guess. As a reference, the default shortcut to open notification center on Windows is <kbd>Meta + N</kbd>. cc @meiyixiang

**Comment by meiyixiang** [comment]
I have fed back your suggestions to relevant products
